book/quick_tour.md

```diff
@@ -1,53 +1,227 @@
+
+```nu
+ls | where size > 10kb
+# => ╭───┬─────────────────┬──────┬───────────┬───────────────╮
+# => │ # │      name       │ type │   size    │   modified    │
+# => ├───┼─────────────────┼──────┼───────────┼───────────────┤
+# => │ 0 │ CONTRIBUTING.md │ file │  11.0 KiB │ 5 months ago  │
+# => │ 1 │ Cargo.lock      │ file │ 194.6 KiB │ 2 minutes ago │
+# => │ 2 │ README.md       │ file │  12.0 KiB │ 16 hours ago  │
+# => │ 3 │ toolkit.nu      │ file │  20.0 KiB │ 16 hours ago  │
+# => ╰───┴─────────────────┴──────┴───────────┴───────────────╯
+```
+
+## More Than Just Directories
+
+Of course, this isn't limited to the `ls` command. Nushell follows the Unix philosophy where each command does one thing well and you can typically expect the output of one command to become the input of another. This allows us to mix-and-match commands in many different combinations.
+
+Let's look at a different command:
+
+```nu:no-line-numbers
+ps
+# => ╭───┬──────┬──────┬───────────────┬──────────┬──────┬───────────┬─────────╮
+# => │ # │ pid  │ ppid │     name      │  status  │ cpu  │    mem    │ virtual │
+# => ├───┼──────┼──────┼───────────────┼──────────┼──────┼───────────┼─────────┤
+# => │ 0 │    1 │    0 │ init(void)    │ Sleeping │ 0.00 │   1.2 MiB │ 2.2 MiB │
+# => │ 1 │    8 │    1 │ init          │ Sleeping │ 0.00 │ 124.0 KiB │ 2.3 MiB │
+# => │ 2 │ 6565 │    1 │ SessionLeader │ Sleeping │ 0.00 │ 108.0 KiB │ 2.2 MiB │
+# => │ 3 │ 6566 │ 6565 │ Relay(6567)   │ Sleeping │ 0.00 │ 116.0 KiB │ 2.2 MiB │
+# => │ 4 │ 6567 │ 6566 │ nu            │ Running  │ 0.00 │  28.4 MiB │ 1.1 GiB │
+# => ╰───┴──────┴──────┴───────────────┴──────────┴──────┴───────────┴─────────╯
+```
+
+You may be familiar with the Linux/Unix `ps` command. It provides a list of all of the current processes running in the system along with their current status. As with `ls`, Nushell provides a cross-platform, built-in [`ps` command](/commands/docs/ps.md) that returns its results as structured data.
+
+::: note
+The traditional Unix `ps` only shows the current process and its parents by default. Nushell's implementation shows all of the processes on the system by default.
+:::
+
+What if we wanted to just show the processes that are actively running? As with `ls` above, we can also work with the table that `ps` _outputs_:
+
+```nu
+ps | where status == Running
+# => ╭───┬──────┬──────┬──────┬─────────┬──────┬──────────┬─────────╮
+# => │ # │ pid  │ ppid │ name │ status  │ cpu  │   mem    │ virtual │
+# => ├───┼──────┼──────┼──────┼─────────┼──────┼──────────┼─────────┤
+# => │ 0 │ 6585 │ 6584 │ nu   │ Running │ 0.00 │ 31.9 MiB │ 1.2 GiB │
+# => ╰───┴──────┴──────┴──────┴─────────┴──────┴──────────┴─────────╯
+```
+
+::: tip
+Remember above, where the `size` column from the `ls` command was a `filesize`? In this case, `status` may look like an "enum" type from some programming languages,
```

Comment:
Not sure if I would raise this expectation when we can not deliver on it yet 😅

Reply:
Lol - Fair enough.  Don't whet the appetite for something we can't deliver.